modules/installation/pages/general-requirements.adoc

```diff
@@ -72,3 +72,41 @@ In order to use the {webui} to manage your {susemgr} environment, you will need
 
 Windows Internet Explorer is not supported.
 The {productname} {webui} will not render correctly under Windows Internet Explorer.
+
+
+
+== Partition Permissions
+
+When you create disk partitions for the {productname} Server and Proxy, ensure you set the permissions correctly.
+
+For [path]``/var/lib/pgsql``:
+
+* Owner: Read, Write, Execute
```

Comment:
Can wen name the owner and group here as well? It is important that the user and group name is correct.